test/test_nn.py

```diff
@@ -1556,6 +1556,73 @@ def test_module_to_argparse(self):
         with self.assertRaises(TypeError):
             net.to(cpu, torch.tensor(3, dtype=torch.long), non_blocking=True)
 
+    @unittest.skipIf(not TEST_CUDA, 'CUDA not available')
+    def test_global_flag_overwrite_module_params_on_conversion(self):
+        torch.__future__.set_overwrite_module_params_on_conversion(False)
+
+        # yf225 TODO: add comment for this test! Saying that the current default behavior is just wrong
```

Comment:
nit: I'm not really a fan of putting usernames in comments; if anyone really cares they can look at the history.